magenta-lib/src/main/scala/magenta/tasks/SetStackPolicyTask.scala

```diff
@@ -12,20 +12,58 @@ case object DenyReplaceDeletePolicy extends StackPolicy
 
 object StackPolicy {
 
-  def toPolicyDoc(policy: StackPolicy, sensitiveResourceTypes: Set[String], accountPrivateTypes: () => Set[String]): String = policy match {
-    case AllowAllPolicy =>
-      ALLOW_ALL_POLICY
-    case DenyReplaceDeletePolicy =>
-      val sensitiveResources = sensitiveResourceTypes.filter(t => t.startsWith("AWS") || accountPrivateTypes().contains(t))
-      DENY_REPLACE_DELETE_POLICY(sensitiveResources)
+  def toPolicyDoc(policy: StackPolicy, accountResourceTypes: () => Set[String]): String = policy match {
```

Comment:
`sensitiveResourceTypes` argument is [dropped](https://github.com/guardian/riff-raff/pull/874/commits/290d11a6e14c46b375002725744f39d5a16e68be) as the same value was always passed in.